EnglishTranslation-master/ETC.tsv

```diff
@@ -23590,7 +23590,7 @@ ETC_20160811_023584	For the war against demons	마족과의 전쟁을 위해서
 ETC_20160811_023585	To show the might of the Royal Family	왕실의 권위를 나타내려고				
 ETC_20160811_023586	Medzio Diena has been stopped	신수의 날을 완벽하게 막아냈다				
 ETC_20160811_023587	It does not have any relation to Medzio Diena	신수의 날과는 관계가 없다				
-ETC_20160811_023588	Goddess Laima was not strong enough to stop Medzio Diena	제대로 작동했지만 신수의 힘이 너무 강력해서 막지 못했다				
+ETC_20160811_023588	The plan succeeded, but wasn't enough to stop Medzio Diena	제대로 작동했지만 신수의 힘이 너무 강력해서 막지 못했다				
```

Comment:
why removing the text about laima? it changes the context of the line

Reply:
Laima was never mentioned in the native korean line. 

"Laima's plan succeeded, but wasn't enough to stop medzio diena" doesn't make sense in this case, also it's an edit of an edit - since the quest line at this point infers that laima failed her task and leads the player to selecting this choice (or you guess that laima is a demon). you can cross examine it with the Korean line that is at the end of; ```ETC_20160811_023588```